### DISCUSSION TITLE
Communication Problem

### DISCUSSION THREAD

**Discussion by Sidracim** [Yogui79/IntexPureSpa#72] [Q&A]
Hello,

I have a problem with the Pump communication. After the Start of the ESP32 everything looks fine. I get connection to the WLAN an also DATA from the Pump for eg. the Temperature. In the MQTT Explorer also the IntexSpa is visible with the right data. But after a few seconds the communication between the ESP32 and the pumps seem to be broken. I get only Messages like this from the debug console.

First:
IntexSpa/Communication with pump 1

after few Seconds

Debug Pump data : 38 00 00 00 01 15 15 25 00 00 00 00 00 00 00 6D 90
Debug Pump data : 38 00 00 00 01 15 15 25 00 00 00 00 00 00 00 6D 90
Debug Pump data : 38 00 00 00 01 15 15 25 00 00 00 00 00 00 00 6D 90
Debug Pump data : 38 00 00 00 01 15 15 25 00 00 00 00 00 00 00 6D 90
Debug Pump data : 38 00 00 00 01 15 15 25 00 00 00 00 00 00 00 6D 90
Debug Pump data : 38 00 00 00 01 15 15 25 00 00 00 00 00 00 00 6D 90
Debug Pump data : 38 00 00 00 01 15 15 25 00 00 00 00 00 00 00 6D 90
Debug Pump data : 38 00 00 00 01 15 15 25 00 00 00 00 00 00 00 6D 90
Debug Pump data : 38 00 00 00 01 15 15 25 00 00 00 00 00 00 00 6D 90
Debug Pump data : 38 00 00 00 01 15 15 25 00 00 00 00 00 00 00 6D 90
Debug Pump data : 38 00 00 00 01 15 15 25 00 00 00 00 00 00 00 6D 90
Debug Pump data : 38 00 00 00 01 15 15 25 00 00 00 00 00 00 00 6D 90
Debug Send command : 38 00 00 00 00 00 22 CE

and so on

Have anyone an Idea?

**Comment by rommess** [comment]
Hi,
How close your esp32 to the pump is ? Mine is 5 meters maximum.
Also make sure the remote control is on the pump to prevent sleep mode.

**Reply by Sidracim** [reply]
Hi,
the ESP is aprox 30cm away from the pump the display is always on the pump unit. 
I checked with Mqtt explorer my system and found out that the pool isnt permanent online. Is this normal. The logfiles from my accesspoint shows that esp is permanent online.
